lib/led/digits.js

```diff
@@ -0,0 +1,13 @@
+var LedControl = require("./ledcontrol"),
+  _ = require("lodash");
+
+// stub implementation; extract functionality from ledcontrol.js
+var Digits = function(opts) {
+  opts.isMatrix = false;
+  return new LedControl(opts);
+};
+_.extend(Digits, LedControl, {
```

Comment:
Almost never, but...

>  If we just need one or two functions, can we cherry-pick?

I guess? I haven't had much use for it in 3 years. Let's take it case by case.